src/test/sql/sql_translator_test.cpp

```diff
@@ -676,4 +676,26 @@ TEST_F(SQLTranslatorTest, ColumnAlias) {
   EXPECT_EQ(*expressions[1]->alias(), std::string("z"));
 }
 
+TEST_F(SQLTranslatorTest, MultipleJoinConditionsOnLeftSide) {
```

Comment:
I'd include a complex case as well, with multiple predicates on both sides. Also, consider the cool LQP comparisons as in 

```c++
TEST_F(SQLTranslatorTest, SelectInnerJoin) {
  const auto query = "SELECT * FROM table_a AS a INNER JOIN table_b AS b ON a.a = b.a;";
  auto result_node = compile_query(query);

  _stored_table_node_a->set_alias("a");
  _stored_table_node_b->set_alias("b");

  // clang-format off
  const auto projection_node =
  ProjectionNode::make_pass_through(
    JoinNode::make(JoinMode::Inner, LQPColumnReferencePair{_table_a_a, _table_b_a}, PredicateCondition::Equals,
      _stored_table_node_a,
      _stored_table_node_b));
  // clang-format on

  EXPECT_LQP_EQ(projection_node, result_node);
}
```

for this, they are neat! And way better at verifying the correct LQP is generated.